GitCommands/Settings/AppSettings.cs

```diff
@@ -470,6 +470,8 @@ public static bool ShowResetAllChanges
         public static ISetting<bool> ShowConEmuTab { get; } = Setting.Create(DetailedSettingsPath, nameof(ShowConEmuTab), true);
         public static ISetting<string> ConEmuStyle { get; } = Setting.Create(DetailedSettingsPath, nameof(ConEmuStyle), "<Solarized Light>");
         public static ISetting<string> ConEmuTerminal { get; } = Setting.Create(DetailedSettingsPath, nameof(ConEmuTerminal), "bash");
+        public static ISetting<bool> ProcessHistoryAsTab { get; } = Setting.Create(DetailedSettingsPath, nameof(ProcessHistoryAsTab), true);
```

Comment:
Maybe false for now, until there is more info?

Reply:
This settings toggles between the Tab and the docked Panel.
If we really would want to hide it, the default of `ProcessHistoryDepth` would be set to `0`.

> until there is more info

This shall never replace the Git Command Log. I want to have the recent interactive git output at hand (with clickable links a.o.), and a trace of those exceptions and hints which shall not be presented using modal popup.
Please elaborate what you think was missing here.